### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by loftusa**
I updated the README to include a brief description about this project and links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # chemnlp
-ChemNLP project
+
+ChemNLP is a project that aims to create an extensive chemistry dataset and use it to train large language models (LLMs) for a wide range of chemistry applications. The resulting models will be capable of predicting properties of materials and molecules, chemical reactions, and even generating computational chemistry code. The potential use cases include accelerating drug discovery, reducing the need for animal experiments, and bringing machine learning to more complex tasks in chemistry. The project builds on prior work that has shown basic chemistry capabilities in LLMs and will use a model setup similar to Med-PaLM and "instruction prompt tuning" (IPT) to fine-tune the LLMs with a few trainable parameters.
+
+## Relevant Links
+**Project Proposal**: https://docs.google.com/document/d/1C44EKSJRojm39P2CaxnEq-0FGwDRaknKxJ8lZI6xr5M/edit?usp=sharing
```
Comment (MicPie) [inline README.md:6]: To make the links look nicer use [link description](https://xy.com/), see https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links for more details.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
 # chemnlp
```
Comment (MicPie) [inline README.md:1]: ChemNLP

File: README.md [hunk 3]
```diff
@@ -1,2 +1,9 @@
...
-ChemNLP project
+
+ChemNLP is a project that aims to create an extensive chemistry dataset and use it to train large language models (LLMs) for a wide range of chemistry applications. The resulting models will be capable of predicting properties of materials and molecules, chemical reactions, and even generating computational chemistry code. The potential use cases include accelerating drug discovery, reducing the need for animal experiments, and bringing machine learning to more complex tasks in chemistry. The project builds on prior work that has shown basic chemistry capabilities in LLMs and will use a model setup similar to Med-PaLM and "instruction prompt tuning" (IPT) to fine-tune the LLMs with a few trainable parameters.
```
Comment (MicPie) [inline README.md:3]: Maybe remove this sentence until we have some hard data to back that up:
"The resulting models will be capable of predicting properties of materials and molecules, chemical reactions, and even generating computational chemistry code."

File: README.md [hunk 4]
```diff
@@ -1,2 +1,9 @@
...
+
+## Relevant Links
+**Project Proposal**: https://docs.google.com/document/d/1C44EKSJRojm39P2CaxnEq-0FGwDRaknKxJ8lZI6xr5M/edit?usp=sharing
+**Chemistry datasets repo**: https://github.com/kjappelbaum/awesome-chemistry-datasets
+**Github task board**: https://github.com/orgs/OpenBioML/projects/5/views/1
```
Comment (MicPie) [inline README.md:8]: I just added a dataset board too: https://github.com/orgs/OpenBioML/projects/6/views/1

**Comment by MicPie** [comment]
Hi @loftusa I just left some small comments. Just let me know if you have time to change the few things. No worries if not, then I can do it. Just tell me. 👍

**Review by MicPie — CHANGES_REQUESTED** [review]
Hi @loftusa I just left some small comments. Just let me know if you have time to change the few things. No worries if not, then I can do it. Just tell me. 😃

File: README.md [hunk 5]
```diff
@@ -1,2 +1,9 @@
 # chemnlp
-ChemNLP project
+
+ChemNLP is a project that aims to create an extensive chemistry dataset and use it to train large language models (LLMs) for a wide range of chemistry applications. The resulting models will be capable of predicting properties of materials and molecules, chemical reactions, and even generating computational chemistry code. The potential use cases include accelerating drug discovery, reducing the need for animal experiments, and bringing machine learning to more complex tasks in chemistry. The project builds on prior work that has shown basic chemistry capabilities in LLMs and will use a model setup similar to Med-PaLM and "instruction prompt tuning" (IPT) to fine-tune the LLMs with a few trainable parameters.
+
```
Comment (kjappelbaum) [inline README.md:4]: ## Contributing
ChemNLP is an open-source project - your involvement is warmly welcome! If you're excited to join us, we recommend the following steps:

- 🤔 Looking for ideas? See our [Task Board](https://github.com/orgs/OpenBioML/projects/5/views/1) to see what we may need help with.

- 💡 Have an idea? Create an issue

Feel free to join our [Discord](https://t.co/YMzpevmkiN)!

Comment (kjappelbaum) [reply README.md:4]: Let's add then the CoC and Contribution Guide later #6

File: README.md (hunk 4 above)
Comment (MicPie) [reply README.md:8]: we don't use that anymore

**Comment by MicPie** [comment]
Closed this PR as it is superseded by PR https://github.com/OpenBioML/chemnlp/pull/21.

**Pull request closed without merging**